lightningd/pay.c

```diff
@@ -235,7 +235,7 @@ json_sendpay_in_progress(struct command *cmd,
 {
 	struct json_stream *response = json_stream_success(cmd);
 	json_add_string(response, "message",
-			"Monitor status with listpayments or waitsendpay");
+			"Monitor status with listsendpays or waitsendpay");
```

Comment:
No, `listsendpays` is wrong in this context.  Replace with `listpays` instead!  `listsendpays` will sometimes waffle a payment as "failed", because `pay` works by brute-forcing `sendpay`s.  `listpays` filters that out and sets a payment as "ongoing" as long as `pay` is still brute-forcing it.

Reply:
People have lost funds running services which use `pay` to pay to clients but monitor the success using `listsendpays`, so we should really ***not*** point to `listsendpays`, just `listpays`.

Reply:
Updated in 8c88e2cf.